keyboards/handwired/plonk/info.json

```diff
@@ -0,0 +1,18 @@
+{
+    "keyboard_name": "Plonk",
+    "url": "https://imgur.com/gJLgwQI",
+    "maintainer": "Kevin Lockwood",
+    "width": 12,
+    "height": 4,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"label":"k00", "x":0, "y":0},
+                {"label":"k01", "x":1, "y":0},
+                {"label":"k02", "x":2, "y":0},
+                {"label":"k10", "x":0, "y":1, "w":1.5},
+                {"label":"k12", "x":1.5, "y":1, "w":1.5}
+            ]
+        }
```

Comment:
This needs updating, to work on the Configurator. Copy this section here from layouts/ortho_4x12/info.json.

Reply:
WIll do!

Reply:
This is the first time I've done this, so thank you for helping me through it all. I think I got it to work.